tcms/testruns/models.py

```diff
@@ -66,25 +69,21 @@ def __str__(self):
 
     @classmethod
     def to_xmlrpc(cls, query=None):
-        from tcms.xmlrpc.serializer import TestRunXMLRPCSerializer
-        from tcms.xmlrpc.utils import distinct_filter
-
         _query = query or {}
         qs = distinct_filter(TestRun, _query).order_by('pk')
         serializer = TestRunXMLRPCSerializer(model_class=cls, queryset=qs)
         return serializer.serialize_queryset()
 
     @classmethod
     def list(cls, query):
-        conditions = []
-
         mapping = {
             'search': lambda value: Q(run_id__icontains=value) | Q(summary__icontains=value),
             'summary': lambda value: Q(summary__icontains=value),
             'product': lambda value: Q(build__product=value),
             'product_version': lambda value: Q(product_version=value),
             'plan': lambda value:
-            Q(plan__plan_id=int(value)) if is_int(value) else Q(plan__name__icontains=value),
+                    Q(plan__plan_id=int(value))
+                    if is_int(value) else Q(plan__name__icontains=value),  # noqa: E131
```

Comment:
Move this thing into a small helper function declared above the class definition.